src/urllib3/poolmanager.py

```diff
@@ -311,6 +313,15 @@ def connection_from_context(
         ``request_context`` must at least contain the ``scheme`` key and its
         value must be a key in ``key_fn_by_scheme`` instance variable.
         """
+        if "implicit_scheme" in request_context:
+            warnings.warn(
+                "Schemeless URL are deprecated and will be unsupported "
```

Comment:
To avoid the use of "Schemeless" (which might be tough to parse for some folks) let's opt for something like:

"URLs without a scheme (ie 'https://') are deprecated and begin raising an error in a future version of urllib3. To avoid this DeprecationWarning ensure all URLs start with 'https://' or 'http://'. Read more in this issue: ..."

Reply:
Done ✔️